### PR TITLE
Update .NET glossary

### PR DESCRIPTION
**Pull request by tdykstra**

### CODE REVIEW

File: docs/standard/glossary.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Base Class Library.
 
 A set of libraries that comprise the System.\* (and to a limited extent Microsoft.\*) namespaces. The BCL is a general purpose, lower-level framework that higher-level application frameworks, such as ASP.NET Core, build on.
 
-The source code of the BCL for [.NET 5 (and .NET Core) and later versions](#net-5-and-later-versions) is contained in the [.NET runtime repository](https://github.com/dotnet/runtime). The majority of these BCL APIs are also available in .NET Framework, so you can think of this source code as a fork of the .NET Framework BCL source code.
+The source code of the BCL for [.NET 5 (and .NET Core) and later versions](#net-5-and-later-versions) is contained in the [.NET runtime repository](https://github.com/dotnet/runtime). Most these BCL APIs are also available in .NET Framework, so you can think of this source code as a fork of the .NET Framework BCL source code.
```
Comment (gewarren) [inline docs/standard/glossary.md:56]:
```suggestion
The source code of the BCL for [.NET 5 (and .NET Core) and later versions](#net-5-and-later-versions) is contained in the [.NET runtime repository](https://github.com/dotnet/runtime). Most of these BCL APIs are also available in .NET Framework, so you can think of this source code as a fork of the .NET Framework BCL source code.
```

File: docs/standard/glossary.md [hunk 2]
```diff
@@ -175,19 +177,22 @@ See the [Mono documentation](https://www.mono-project.com/docs/).
 
 ## .NET
 
-The umbrella term for [.NET Standard](#net-standard) and all [.NET implementations](#implementation-of-net) and workloads. Always fully capitalized, never ".Net".
+* In general, *.NET* is the umbrella term for [.NET Standard](#net-standard) and all [.NET implementations](#implementation-of-net) and workloads.
+* More specifically, .NET refers to the implementation of .NET that is recommended for all new development: [.NET 5 (and .NET Core) and later versions](#net-5-and-later-versions).
+
+Fo example, the first meaning is intended in phrases such as "implementations of .NET." The second meaning is intended in names such as [.NET SDK](#net-sdk) and [.NET CLI](#net-cli).
```
Comment (gewarren) [inline docs/standard/glossary.md:183]:
```suggestion
For example, the first meaning is intended in phrases such as "implementations of .NET." The second meaning is intended in names such as [.NET SDK](#net-sdk) and [.NET CLI](#net-cli).
```

File: docs/standard/glossary.md [hunk 3]
```diff
@@ -213,7 +218,7 @@ Compilation happens on the developer's machine similar to the way a C++ compiler
 
 UWP was the first application framework supported by .NET Native. Now, we support building native console apps for Windows, macOS, and Linux.
```
Comment (gewarren) [inline docs/standard/glossary.md:219]: I think UWP is the only place it's currently used/supported.

**Review by gewarren — APPROVED** [review]
Looks good!

File: docs/standard/glossary.md (hunk 3 above)
Comment (tdykstra) [reply docs/standard/glossary.md:219]:
```suggestion
UWP is the application framework supported by .NET Native.
```

**Comment by tdykstra** [comment]
Thanks for the review, @gewarren !

**Pull request merged**